playbooks/provider/lago/LagoInitFile.yml.j2

```diff
@@ -6,6 +6,7 @@ nat-settings: &nat-settings
     management: False
 
 vm-common-settings: &vm-common-settings
+    vCpu: 4
```

Comment:
it is lowercased `vcpu`, I am afraid that it might cause that this option is going to ignored :-O

Reply:
@lukas-bednar looks like it doesn't work :(

Reply:
Forget to say, your latest suggestion works